src/Header/Title/Title.js

```diff
@@ -0,0 +1,8 @@
+const Title = () => {
+    return (
+        <div className={'px-3 py-3'}>
```

Comment:
this padding is probably going to be on the header itself rather than the title (that way the padding is on all sides of the header if you get what i mean)

Reply:
done